src/gui/fspathedit.cpp

```diff
@@ -196,6 +196,8 @@ FileSystemPathEdit::FileSystemPathEdit(Private::FileEditorWithCompletion *editor
     layout->addWidget(d->m_browseBtn);
 
     connect(d->m_browseAction, &QAction::triggered, this, [this]() { this->d_func()->browseActionTriggered(); });
+
+    setFocusProxy(editor->widget());
```

Comment:
Please move this line after `editor->widget()->setParent(this);` so that similar variables group together.
Also Please squash your commits into one.

Reply:
Done.